tensorflow/tools/pip_package/BUILD

```diff
@@ -36,11 +36,15 @@ transitive_hdrs(
         "//tensorflow/cc/saved_model:reader",
         "//tensorflow/cc/saved_model:bundle_v2",
         "//tensorflow/c/experimental/filesystem:filesystem_interface",
+        "//tensorflow/c/experimental/next_pluggable_device:c_api_hdrs",
         "//tensorflow/c/experimental/pluggable_profiler:pluggable_profiler_hdrs",
         "//tensorflow/c/experimental/stream_executor:stream_executor_hdrs",
         "//tensorflow/c/experimental/grappler:grappler_hdrs",
         "//tensorflow/c:kernels_hdrs",
         "//tensorflow/c:ops_hdrs",
+        "//tensorflow/compiler/xla/pjrt/c:pjrt_c_api_hdrs",
+        "//tensorflow/compiler/xla/pjrt/c:pjrt_c_api_tpu_hdrs",
```

Comment:
Just to double check, is pjrt_c_api_tpu_hdrs required?

Reply:
@jyingl3 Thanks for the review, pjrt_c_api_tpu.h contains the declaration of GetPjrtApi, which is entry point of plugin loaded by TF
```
 #include "tensorflow/compiler/xla/pjrt/c/pjrt_c_api.h"

 #ifdef __cplusplus
 extern "C" {
 #endif

 // TODO(b/261916900): this will cause symbol collision when we wanna statically
 // link in multiple plugins. We may need to make the function name configurable,
 // or we could add external-only wrapper shims
 // Does not pass ownership of returned PJRT_Api* to caller.
 const PJRT_Api* GetPjrtApi();

 #ifdef __cplusplus
 }
 #endif
```

Reply:
Thanks Zhoulong. We probably should move the declaration of GetPjrtApi or rename this file then. Meanwhile, I do not want to block this change so probably we can go ahead with the it and refactor later. @chuanhaozhuge Does it look good to you?

Reply:
Discussed with @stellaraccident. We think the plugin will likely have its own declaration of GetPjrtApi to add attributes to control/export/etc. And we will add the function typedef with documentation on the name (GetPjrtApi) to pjrt_c_api.h as a guidance.

In this case, could we not expose "//tensorflow/compiler/xla/pjrt/c:pjrt_c_api_tpu_hdrs" here, and let the plugin to has its own declaration of GetPjrtApi?

Reply:
@jyingl3  thanks for the suggestion, I removed the pjrt_c_api_tpu_hdrs, we are ok to declare the GetPjRtApi in plugin side.
Thanks

Reply:
Thanks Zhoulong! I will approve it. May pull in other reviewer if it increases the size too much.